python/cudf/udf_cpp/strings/include/cudf/strings/udf/udf_apis.hpp

```diff
@@ -57,6 +58,21 @@ std::unique_ptr<rmm::device_buffer> to_string_view_array(cudf::column_view const
 std::unique_ptr<cudf::column> column_from_udf_string_array(udf_string* d_strings,
                                                            cudf::size_type size);
 
+/**
+ * @brief Return a STRINGS column given an array of managed_udf_string objects
+ *
+ * This will make a copy of the strings in managed_strings in order to build
+ * the output column.
+ * The individual managed_udf_strings are also cleared freeing each of their internal
```

Comment:
I think the docstring here says that the `managed_udf_string`s are cleared, but it doesn't look like the implementation does that. Am I misunderstanding the code or is the docstring inaccurate?

Reply:
You're right. The freeing functions clear these not the transform functions. I updated this.